multi-node/aws/pkg/config/templates/cloud-config-worker

```diff
@@ -13,19 +13,39 @@ coreos:
             [Unit]
             Requires=flanneld.service
             After=flanneld.service
+            [Service]
+            ExecStart=
+            ExecStart=/usr/lib/coreos/dockerd daemon --host=fd:// $DOCKER_OPTS $DOCKER_CGROUPS $DOCKER_OPT_MTU
 
     - name: kubelet.service
       enable: true
       command: start
       content: |
+{{ if eq .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=docker.service load-rkt-stage1.service
+        After=docker.service load-rkt-stage1.service
```

Comment:
I think I just left in what was already there, then it got removed in master and survived through rebasing or something. Will remove.